### PR TITLE
LPD-42973  Add organization icon to React Flow Edge

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-42972)
Please review from 360cd665628e8b06ea88da2c0c2c173390d8c09b

### CODE REVIEW

**Comment by igor-franca** [comment]
Hi @rebsilva, when implementing these changes in my branch I hadn't noticed a difference in usability, when combined with the munchies changes I'm not sure if I noticed a difference in dragging a node with a relationship in the diagram. Can you take this test and tell me if you had the same feeling? I'm still not sure if the problem is my trackpad

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/ObjectRelationshipEdge.tsx [hunk 1]
```diff
@@ -15,6 +15,9 @@ const labelStyle = {
 	fill: '#FFF',
 	fontSize: '12px',
 	fontWeight: 600,
+	paddingBottom: 1,
+	paddingTop: 1,
+	"white-space": "nowrap",
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/ObjectRelationshipEdge.tsx:20]: Hey Rebecca, I can't set the whitespace using the "whiteSpace" pattern. 
There is a type in typescript that defines the whiteSpece and I can't import it, one approach was to pass the key as a string, I can do this for all the keys so they follow the same pattern, what do you think? 

If you're thinking of something different, let me know :D

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/SimpleObjectRelationshipEdge.tsx [hunk 2]
```diff
@@ -60,25 +73,33 @@ export function SimpleObjectRelationshipEdge({
...
-							selectedObjectRelationshipId: id,
-						},
-						type: TYPES.SET_SELECTED_OBJECT_RELATIONSHIP_EDGE,
-					});
-				}}
-				x={edgeCenterX}
-				y={edgeCenterY}
-			/>
-		</g>
+			<foreignObject
```
Comment (igor-franca) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/SimpleObjectRelationshipEdge.tsx:76]: This approach was based on this [example](https://v9.reactflow.dev/examples/edge-with-button)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/ObjectRelationshipEdge.tsx (hunk 1 above)
Comment (igor-franca) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/ObjectRelationshipEdge.tsx]: Hey Rebeca, the SF changes didn't work very well, I changed things a little

**Comment by igor-franca** [comment]
Rebeca, fell free to review again. Thanks for noticing the misaligned icon! :-team-work-:

**Comment by rebsilva** [comment]
@igor-franca no problem igor! could you please take a look at the conflict? thanks!

**Comment by igor-franca** [comment]
Conflict resolved, final test:

https://github.com/user-attachments/assets/6df729fd-d45b-4200-97c6-a7a401b47ea4

**Review by rebsilva — APPROVED** [review]
LGTM! :cherry_blossom:

**Comment by rebsilva** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:test:object

**Comment by igor-franca** [comment]
ci:report:89701620

**Comment by igor-franca** [comment]
Faults are related, locators need to be updated

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:report:90406881

**Comment by igor-franca** [comment]
Locator successfully corrected, other failures are not related

**Comment by igor-franca** [comment]
@rebsilva, can you take a look at the new commit added?

**Comment by igor-franca** [comment]
The failure in SF is not related, although it falls within the scope of our team

**Comment by igor-franca** [comment]
Manually sent forward [here](https://github.com/brianchandotcom/liferay-portal/pull/156971)

**Pull request closed without merging**